src/Styling/Home.css

```diff
@@ -111,6 +112,7 @@
   color: #fff;
   border: 1px solid #fff;
   cursor: pointer;
+  z-index: 1;
```

Comment:
z-index should only need to be used in the case an element is overlayed in front of or behind another element. in this case, this button is not doing either and should not need a z-index. 
Be careful of your use of z-index. 
Even though it works, you should think about whether or not your implementation is the best one...this goes for your entire app as you develop. not just this element.